### PR TITLE
Implement ROS

### PR DESCRIPTION
**Pull request by andreasHovaldt**

### CODE REVIEW

**Review by TobiasAxelsen — COMMENTED** [review]
shit det bare fin du!

**Review by TheMipmap — COMMENTED** [review]
Looks fine to me

File: Kinect/locate_hand_no_background_kinect.py [hunk 1]
```diff
@@ -131,6 +151,14 @@ def get_hsv_data(img):  #Function to obtain hsv-data from an image.
                         #translation_vector = np.append(translation_vector, [1])
                         
                         print(f"Kinect to hand vector {kinect_to_hand_vector} \n UR to hand vector in ur coor {ur_to_hand_vector} ")
+                        
+                        position_msg.pos_x = ur_to_hand_vector[0]
+                        position_msg.pos_y = ur_to_hand_vector[1]
+                        position_msg.pos_z = ur_to_hand_vector[2]
```
Comment (Shenwei1234) [inline Kinect/locate_hand_no_background_kinect.py:157]: We do a lil shithousery

**Review by Shenwei1234 — COMMENTED** [review]
Shenwei stamp of approval

**Pull request merged**